packages/site/src/components/ContentLoader.tsx

```diff
@@ -12,8 +12,15 @@ export const ContentLoader = () => {
   const { name } = useParams<{ name: string }>();
   const location = useLocation();
 
-  if (location.pathname.includes("/design")) {
-    type = "design";
+  switch (true) {
+    case location.pathname.startsWith("/design"):
```

Comment:
Had to use `.startsWith` instead of `.includes` here because the Design changelog, which also included `/design` wasn't loading and types were getting mixed up